copyhtml/clipboard.ts

```diff
@@ -0,0 +1,12 @@
+export function setClipboardContent(text: string) {
+  (navigator as unknown as {
+    clipboard: { writeText: (text: string) => Promise<void> };
+  }).clipboard
+    .writeText(text)
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      console.log("Copied to clipboard.", { text });
+    });
```

Comment:
Only log success upon non-failure.
```suggestion
    .then(() => {
      console.log("Copied to clipboard.", { text });
    });
    .catch((err) => {
      console.error(err);
    })
```

Reply:
Resolved by <https://github.com/EthanThatOneKid/copyhtml/pull/1/commits/4cc25a7b54ce6437b7b9b6b2d1fc42942e347a96>.

Reply:
Resolved by <https://github.com/EthanThatOneKid/copyhtml/pull/1/commits/c3b0e358f8b3bb7a24420c349abdb36380e7247c>.